### PR TITLE
Refresh the noisy quality report generation according to the paper rush decisions

### PR DESCRIPTION
**Pull request by warenlg**
Partially fix https://github.com/src-d/style-analyzer/issues/333

### CODE REVIEW

File: lookout/style/format/benchmarks/quality_report_noisy.py [hunk 1]
```diff
@@ -268,34 +259,39 @@ def plot_curve(repositories: Iterable[str], recalls: Mapping[str, numpy.ndarray]
...
-                label="input precision threshold: %.2f" % (precision_threshold))
-    handles, labels = ax.get_legend_handles_labels()
-    ax.legend(handles, labels, loc="best", fontsize=17)
-    ax.set_title("Precision-recall curves on the artificial noisy dataset", fontsize=17)
-    ax.set_ylabel("Precision", fontsize=17, labelpad=15)
-    ax.set_xlabel("Recall", fontsize=17, labelpad=15)
-    ax.spines["right"].set_visible(False)
-    ax.spines["top"].set_visible(False)
-    plt.savefig(path_to_figure)
+        predrs_array = numpy.asarray(predrs[repo])
```
Comment (zurk) [inline lookout/style/format/benchmarks/quality_report_noisy.py]: let's join ax1, ax2 parts under the loop?
```python
f, axes = plt.subplots(2, 1, ... )
...
for ax, y_values, ... in zip(axes, (predrs_array, precisions_array), ...):
    ax.tick_params(labelsize=17, color="gray")
    ...
```

File: lookout/style/format/benchmarks/quality_report_noisy.py [hunk 2]
```diff
@@ -305,45 +301,50 @@ def quality_report_noisy(bblfsh: str, language: str, confidence_threshold: float
            Limit drawn as a red horizontal line on the figure.
     :param dir_output: Path to the output directory where to store the quality report in Markdown \
            and the precision-recall curve in png format.
-    :param repos: Input list of urls or paths to the repositories to analyze. \
-           Should be strings separated by newlines.
-    :param retrain: Set True if you want to force model retraining.
+    :param repos: Input list of urls to the repositories to analyze. \
+           Should be strings separated by newlines. If if is None, \
```
Comment (zurk) [inline lookout/style/format/benchmarks/quality_report_noisy.py]:
```suggestion
           Should be strings separated by newlines. If it is None, \
```

File: lookout/style/format/benchmarks/quality_report_noisy.py [hunk 3]
```diff
@@ -305,45 +301,50 @@ def quality_report_noisy(bblfsh: str, language: str, confidence_threshold: float
...
-                    input_pattern_noisy = os.path.join(git_dir_noisy, "**", "*.js")
-                    model_path = os.path.join(git_dir_noisy, "style-analyzer-model", "model.asdf")
-                else:
-                    input_pattern = os.path.join(repo_path, "**", "*.js")
-                    input_pattern_noisy = os.path.join(repo_path + "_noisy", "**", "*.js")
-                    model_path = os.path.join(repo_path, "model.asdf")
+                log.info("Fetching %s", repo_path)
+                git_dir = os.path.join(tmpdirname, repo)
+                git_dir_noisy = os.path.join(tmpdirname, repo + "_noisy")
+                cmd1 = "git clone --single-branch --branch master %s %s" % (repo_path, git_dir)
```
Comment (zurk) [inline lookout/style/format/benchmarks/quality_report_noisy.py]: it is a good idea to have a repository with two commits in the `data` directory. 
The improvement is not related to current PR, so just a note.

File: lookout/style/format/benchmarks/quality_report_noisy.py [hunk 4]
```diff
@@ -227,34 +227,25 @@ def compute_metrics(changes_count: int, predictions_count: int, true_positive: i
     :param changes_count: Overall number of cases.
     :param predictions_count: Total number of predictions made by the model.
     :param true_positive: Number of positive cases predicted as positive.
-    :return: Precision, recall and F1-score metrics.
+    :return: Prediction rate and precision metrics.
     """
     false_positive = predictions_count - true_positive
-    false_negative = changes_count - predictions_count
+    predr = predictions_count / changes_count
```
Comment (zurk) [inline lookout/style/format/benchmarks/quality_report_noisy.py]: let's name it `pred_rate` or `prediction_rate` everywhere.

**Comment by warenlg** [comment]
@vmarkovtsev any objection to merge this ?

**Comment by vmarkovtsev** [comment]
I was confused with the double Travis CI status, sorry

**Pull request merged**